Dockerfile

```diff
@@ -79,10 +90,32 @@ RUN git clone --depth 1 https://github.com/mongodb/devhub.git snooty-devhub
     && cd snooty-devhub                                                                            \
     && npm install --production
 
+# install redoc fork
+RUN git clone -b ${REDOC_CLI_VERSION} --depth 1 https://github.com/mongodb-forks/redoc.git redoc \
```

Comment:
minor, should we put this URL as a var as well

Reply:
We could. The URL is a lot less likely to change than the Redoc CLI version though. I was also just following conventions for our other git repo clones haha